README.md

```diff
@@ -3,8 +3,6 @@ liblifthttp - The fast asynchronous C++17 HTTP client library
 
 You're using curl? Do you even lift?
 
-Copyright (c) 2017, Josh Baldwin
-
 https://github.com/jbaldwin/liblifthttp
```

Comment:
still have jbaldwin's repo github url

Reply:
this is my private version, I'm thinking you meant to push this to the spotx one?